packages/apps-config/src/api/spec/index.ts

```diff
@@ -18,11 +18,13 @@ import nodle from './nodle';
 import plasm from './plasm';
 import stablePoc from './stable-poc';
 import subsocialNode from './subsocial';
+import crust from './crust';
```

Comment:
Can you please move this import up alphabetical as well? (Just makes maintenance easier since the lists are growing...)

Reply:
Sure, fixed